OWNERS

```diff
@@ -3,22 +3,12 @@
 filters:
   ".*":
     reviewers:
-      - davidvossel
-      - rmohr
-      - stu-gott
-      - booxter
-      - vladikr
-      - slintes
-      - cynepco3hahue
-      - phoracek
-      - SchSeba
-      - mfranczy
-      - dhiller
+      - code-reviewers
     approvers:
-      - slintes
-      - davidvossel
-      - vladikr
-      - rmohr
-      - cynepco3hahue
-      - stu-gott
-      - SchSeba
+      - approvers
+  "tests/.*":
```

Comment:
do we care to separate out unit test owners as well? 

I wonder if `*_test.go` might be a valid pattern

Reply:
I'd actually ask @ILpinto and friends to update the filters according to their needs.
This is merely the initial work to show how it works.